### ISSUE TITLE
main panicked

### ISSUE THREAD

**Issue by reiz** [versioneye/veye-checker#7]
By running this command: 

```
./veye_checker lookup_csv shas.csv -a <APPKEY> -o res.csv
```

I got this Exception: 

```
Going to checkup product by SHA: f7a842bcab55de4d1ee8d524baa571dc12684ef2
Fetching product info by sha...
thread 'main' panicked at 'no entry found for key', /Users/rustbuild/src/rust-buildbot/slave/stable-dist-rustc-mac/build/src/libcore/option.rs:715
note: Run with `RUST_BACKTRACE=1` for a backtrace.
```

**Comment by timgluz** [comment]
Can you run without specifying output file and copy the latest requested SHA or latest lines here

**Comment by reiz** [comment]
I just executed it like this: 

```
./veye_checker lookup_csv shas.csv -a MY_API_KEY
```

And now the only output is: 

```
thread 'main' panicked at 'Missing output file', /Users/rustbuild/src/rust-buildbot/slave/stable-dist-rustc-mac/build/src/libcore/option.rs:715
note: Run with `RUST_BACKTRACE=1` for a backtrace.
```

**Comment by timgluz** [comment]
not sure, what's going on here. i wasnt able to replicate the problem with same input data;
But i've hypotheses that it may be related to the bug in the process of product key encoding, which is now fixed; 

Can you try again? ps: don't forget to pull the newest binary for osx;

and can you try this command too?
```
VERSIONEYE_API_KEY="<YOUR API KEY>" ./veye_checker lookup f7a842bcab55de4d1ee8d524baa571dc12684ef2
```

**Comment by timgluz** [comment]
solved in `0.1.0-RC2`
